.ci/deps.sh

```diff
@@ -99,3 +99,6 @@ fi
 # Tailor (Swift) commands
 curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
 sudo bash install.sh
+
+# FlawFinder will be confused by our python 3 venv
+sudo sed -i 's/env python/python2/' /usr/bin/flawfinder
```

Comment:
`make`ing flawfinder doesn't do anything but the same as the current package installation. The reason this sed is needed is that flawfinder is written with python 2 only systems in mind. It won't work unless you sed it. The ArchLinux AUR package uses a sed expression too for that reason.